.github/workflows/certification.yml

```diff
@@ -245,10 +252,65 @@ jobs:
           exit 1
         fi
 
+    - name: Prepare Cert Coverage Info
+      run: |
+       mkdir -p path/to/cov_files
+       SOURCE_PATH_LINEAR=$(echo ${{ env.SOURCE_PATH }} |sed 's#/#\.#g')
+       echo "${{ env.cover_per_only }}" >> path/to/cov_files/$SOURCE_PATH_LINEAR.txt
+
+    - name: Upload Cert Coverage Info
+      uses: actions/upload-artifact@v3
+      with:
+        name: cert_cov
+        path: path/to/cov_files
+        retention-days: 1
+
+    - name: Discord notification
+      env:
+        DISCORD_WEBHOOK: ${{ secrets.DISCORD_MONITORING_WEBHOOK_URL }}
+      uses: Ilshidur/action-discord@0c4b27844ba47cb1c7bee539c8eead5284ce9fa9
```

Comment:
We are using the same SHA in two places, so it might be better to store it in a variable and then use. In future while changing, only one place will need to be changed.